frontend_tests/puppeteer_tests/compose.ts

```diff
@@ -87,10 +93,13 @@ async function test_reply_with_r_shortcut(page: Page): Promise<void> {
     await page.keyboard.press("KeyK");
     await page.keyboard.press("KeyR");
     await common.check_form_contents(page, "#send_message_form", {
-        stream_message_recipient_stream: "Verona",
         stream_message_recipient_topic: "Reply test",
         content: "",
     });
+    assert.equal(
+        await common.get_text_from_selector(page, "#compose_select_stream_name"),
+        "Verona",
+    );
```

Comment:
I would consider doing a prep commit extracting the `common.check_form_contents` block as a function in the puppeteer library file -- something like `await common.check_compose_state(...)` that checks the stream/topic/content in the correct way -- then you end up just needing to do these hunks inside that common function, and we get a more readable test as a bonus.

Reply:
done in #24133!